Configure.pri.in

```diff
@@ -10,7 +10,7 @@ QMAKE_CXXFLAGS += @DEFS@
 QMAKE_CXXFLAGS += -frounding-math -std=c++11
 
 CONFIG += @HAS_NEWMAT@ @HAS_CPPUNIT@ \
-  @HAS_COVERAGE@ @HAS_SERVICES@ @HAS_STXXL@ @HAS_RND@ @HAS_NODEJS@ @HAS_UI_TESTS@ \
+  @HAS_COVERAGE@ @HAS_SERVICES@ @HAS_STXXL@ @HAS_RND@ @HAS_JOSM@ @HAS_NODEJS@ @HAS_UI_TESTS@ \
```

Comment:
`hoot-josm` is its own project and can be turned off at any time by omitting `--with-josm`